astropy/io/fits/connect.py

```diff
@@ -127,7 +132,7 @@ def read_table_fits(input, hdu=None, astropy_native=False):
 
     else:
 
-        hdulist = fits_open(input)
+        hdulist = fits_open(input, character_as_bytes=True, memmap=memmap)
```

Comment:
From the python3 perspective, this is effectively an API change - you now get back something different than you did before. I think this is fine, but probably good to list it under API changes as well.

Reply:
@mhvk - it shouldn't really impact anyone if the unicode sandwich works properly though, but yes I'll list it in the API changes anyway.

Reply:
I think one shouldn't presume nobody does a `column.view(...)`...

Reply:
Done